evalml/tests/model_understanding_tests/prediction_explanations_tests/test_explainers.py

```diff
@@ -822,3 +827,27 @@ class LinearPipelineDropDates(pipeline_class):
         assert set(explanation['drill_down']['currency']['feature_names']) == EXPECTED_CURRENCY_FEATURES
         assert set(explanation['drill_down']['provider']['feature_names']) == EXPECTED_PROVIDER_FEATURES_OHE
         assert set(explanation['drill_down']['datetime']['feature_names']) == {"datetime_year", "datetime_day_of_week"}
+
+
+@pytest.mark.parametrize("problem_type", [ProblemTypes.BINARY, ProblemTypes.MULTICLASS, ProblemTypes.TIME_SERIES_BINARY, ProblemTypes.REGRESSION])
+def test_explain_predictions_stacked_ensemble(problem_type, dummy_stacked_ensemble_binary_pipeline, dummy_stacked_ensemble_multiclass_pipeline,
+                                              dummy_stacked_ensemble_ts_binary_pipeline, dummy_stacked_ensemble_regressor_pipeline,
+                                              X_y_binary, X_y_multiclass, X_y_regression):
+    if is_binary(problem_type):
+        X, y = X_y_binary
+        if problem_type == ProblemTypes.BINARY:
```

Comment:
nitpick but do we want to use our `is_time_series` if we're going with this pattern heh